csv2xml/csv2xml.py

```diff
@@ -452,7 +452,7 @@ def fix_module_name(path):
     manifest_file = os.path.join(path, '__manifest__.py')
     if os.path.exists(openerp_file) and os.path.isfile(openerp_file):
         pass
-    if os.path.exists(manifest_file) and os.path.isfile(manifest_file):
+    elif os.path.exists(manifest_file) and os.path.isfile(manifest_file):
```

Comment:
@yaniaular 

What do you think about this change:

```
if not (os.path.exists(openerp_file) and os.path.isfile(openerp_file)) and not (os.path.exists(manifest_file) and os.path.isfile(manifest_file)):
    msg = ('The given module is not a openerp module. Missing'
                 ' __openerp__.py or __manifest__.py file.')		                 ' __openerp__.py or __manifest__.py file.')
         raise argparse.ArgumentTypeError(msg)		         raise argparse.ArgumentTypeError(msg)
     return path		     return path
```

To avoid the `pass`?